pytroll_collectors/geographic_gatherer.py

```diff
@@ -90,7 +91,7 @@ def _setup_publisher(self):
     def _setup_triggers(self):
         """Set up the granule triggers."""
         for section in self._config.sections():
-            regions = [get_area_def(region)
+            regions = [parse_area_file(self._config.get(section, 'area_definition_file'), region)[0]
```

Comment:
Should we default to `SATPY_CONFIG_DIR/areas.yaml` to keep some kind of backward compatibility?

Reply:
Ok, I'll add that.